examples/addon/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "whisper-addon",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "author": "Qanhe Chen",
+  "license": "ISC",
```

Comment:
Is `ISC` compatible with `MIT`?
I guess it is ok, but would you mind changing this to `MIT` to keep things simple?